middleware/auth.ts

```diff
@@ -1,5 +1,8 @@
 export default defineNuxtRouteMiddleware(() => {
   const { isLoggedIn } = useAuthStore();
-  if (isLoggedIn) return true;
-  return navigateTo("/login");
+  if (!isLoggedIn) {
```

Comment:
Can't access middleware to check token when refresh/reload the page.

Reply:
bhai don't understand :(